### PR TITLE
Support custom types

### PR DESCRIPTION
**Pull request by rmccue**
I somehow forgot to include this in the first iteration. 🤦

### CODE REVIEW

File: inc/namespace.php [hunk 1]
```diff
@@ -33,7 +33,6 @@ function set_up_checks() {
 	}
 
 	add_filter( 'wp_insert_post_data', __NAMESPACE__ . '\\block_publish_if_failing', 10, 2 );
-	add_filter( 'rest_pre_insert_post', __NAMESPACE__ . '\\block_publish_for_rest', 10, 2 );
```
Comment (kadamwhite) [inline inc/namespace.php:36]: How does removing this relate to the rest of the newly-added logic?

Comment (rmccue) [reply inc/namespace.php:36]: It's moved down into post-init so we can loop over the post types (L172)

Comment (kadamwhite) [reply inc/namespace.php:36]: Aha, thanks, had missed that. Cool!

**Comment by rmccue** [comment]
Oops, need to reimplement the block check though!

**Pull request merged**